purgatory.gemspec

```diff
@@ -2,16 +2,16 @@
 # DO NOT EDIT THIS FILE DIRECTLY
 # Instead, edit Jeweler::Tasks in Rakefile, and run 'rake gemspec'
 # -*- encoding: utf-8 -*-
-# stub: purgatory 3.0.0 ruby lib
+# stub: purgatory 3.1.0 ruby lib
 
 Gem::Specification.new do |s|
   s.name = "purgatory"
-  s.version = "3.0.0"
+  s.version = "3.1.0"
 
   s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
   s.require_paths = ["lib"]
   s.authors = ["Elan Dubrofsky"]
-  s.date = "2014-11-07"
+  s.date = "2017-07-14"
```

Comment:
I don't think we want to change this. Is it not the original launch date? I'm pretty sure we've made changes since 2014.